File: grading-students/grading-students.dart
List<int> gradingStudents(List<int> grades) {
  for (int i = 0; i < grades.length; i++) {
    if (grades[i] >= 38) {
      int nextMultipleOfFive = (grades[i] / 5).ceil() * 5;
      if (nextMultipleOfFive - grades[i] < 3) {
        grades[i] = nextMultipleOfFive;
      }
    }
  }
  return grades;
}

int main() {
    print(gradingStudents([73, 67, 38, 33]));
    return 0;
}

File: grading-students/grading-students.js
function gradingStudents(grades) {
  for (let i in grades) {
    if (grades[i] >= 38) {
      const x = grades[i] % 5;
      if (x >= 3) grades[i] += 5 - x;
    }
  }
  return grades;
}
